src/main/java/jenkins/plugins/slack/SlackNotifier.java

```diff
@@ -470,6 +491,26 @@ public boolean prebuild(AbstractBuild<?, ?> build, BuildListener listener) {
         return super.prebuild(build, listener);
     }
 
+    public MatrixAggregator createAggregator(MatrixBuild matrixBuild, Launcher launcher, BuildListener buildListener) {
+        return new MatrixAggregator(matrixBuild, launcher, buildListener) {
+            @Override
+            public boolean startBuild() throws InterruptedException, IOException {
+                if (getMatrixTriggerMode().forParent) {
+                    return SlackNotifier.this.perform(this.build, this.launcher, this.listener);
+                }
+                return true;
+            }
+
+            @Override
+            public boolean endBuild() throws InterruptedException, IOException {
```

Comment:
IntelliJ says these exception declarations are unneeded?
```suggestion
            public boolean startBuild() {
```